<launch>
    <!-- time -->
    <param name="use_sim_time" value="false" />

    <!-- 线束16 32 64 -->
    <param name="scan_line" type="int" value="64" />
    <param name="scan_fre" type="double" value="0.1" />

    <!-- if 1, do mapping 10 Hz, if 2, do mapping 5 Hz. Suggest to use 1, it will adjust frequence automaticlly -->
    <param name="mapping_skip_frame" type="int" value="1" /> 

    <!-- remove too closed points 雷达最近点滤波 -->
    <param name="minimum_range" type="double" value="0.4"/> 

    <!-- 面点和角点降采样粒度（体素滤波） -->
    <param name="mapping_line_resolution" type="double" value="0.2"/> 
    <param name="mapping_plane_resolution" type="double" value="0.4"/> 

    <!-- lidar topic 雷达话题-->    
    <!-- <param name="pointTopic" type="string" value="/lslidar_point_cloud"/>      -->
    <param name="pointTopic" type="string" value="/kitti/velo/pointcloud"/>     

    <node pkg="a_loam" type="ascanRegistration" name="ascanRegistration" output="screen" />
    <node pkg="a_loam" type="alaserOdometry" name="alaserOdometry" output="screen" />

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find a_loam)/rviz_cfg/aloam_velodyne.rviz" />
    </group>

</launch>


